pkg/minikube/command/kic_runner.go

```diff
@@ -150,22 +151,34 @@ func (k *kicRunner) Copy(f assets.CopyableFile) error {
 		assetName = tmpFile.Name()
 	}
 
-	// based of format of "docker cp containerName:destination"
-	destination := fmt.Sprintf("%s:%s/%s", k.nameOrID, f.GetTargetDir(), f.GetTargetName())
-
 	perms, err := strconv.ParseInt(f.GetPermissions(), 8, 0)
 	if err != nil {
-		return errors.Wrapf(err, "error converting permissions %s to integer", f.GetPermissions())
+		return errors.Wrapf(err, "converting permissions %s to integer", f.GetPermissions())
 	}
 
 	// Rely on cp -a to propagate permissions
 	if err := os.Chmod(assetName, os.FileMode(perms)); err != nil {
 		return errors.Wrapf(err, "chmod")
 	}
-	if out, err := exec.Command(k.ociBin, "cp", "-a", assetName, destination).CombinedOutput(); err != nil {
-		return errors.Wrapf(err, "error copying %s into node, output: %s", f.GetAssetName(), string(out))
+	if k.ociBin == oci.Podman { // Podman cp command doesn't match docker and doesn't have -a
```

Comment:
split out a podman and docker helper function for this.

Reply:
done